gooddata-java/src/test/java/com/gooddata/sdk/service/hierarchicalconfig/HierarchicalConfigServiceIT.java

```diff
@@ -0,0 +1,135 @@
+/*
+ * Copyright (C) 2004-2021, GoodData(R) Corporation. All rights reserved.
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE.txt file in the root directory of this source tree.
+ */
+package com.gooddata.sdk.service.hierarchicalconfig;
+
+import com.gooddata.sdk.model.hierarchicalconfig.SettingItem;
+import com.gooddata.sdk.model.hierarchicalconfig.Settings;
+import com.gooddata.sdk.model.project.Project;
+import com.gooddata.sdk.service.AbstractGoodDataIT;
+import org.springframework.web.util.UriTemplate;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import static com.gooddata.sdk.common.util.ResourceUtils.readObjectFromResource;
+import static com.gooddata.sdk.common.util.ResourceUtils.readStringFromResource;
+import static net.jadler.Jadler.onRequest;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.*;
+import static org.hamcrest.Matchers.is;
+
+public class HierarchicalConfigServiceIT extends AbstractGoodDataIT {
+
+    private HierarchicalConfigService service;
+    private Project project;
+
+    private static final UriTemplate PROJECT_SETTINGS_TEMPLATE = Settings.PROJECT_SETTINGS_TEMPLATE;
```

Comment:
Why not to use static import for such cases directly?

Reply:
Changed